Dshop/apps/products_catalogue/tests.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from django.test import RequestFactory
+from django.test import Client
```

Comment:
No :-) don't use this client. 
Look here please: https://github.com/pymasterspl/Dshop/blob/66a7db0db31483d3fdb7783b934fa9afbef565ac/Dshop/apps/products_catalogue/tests/test_products.py#L29
Please look how the client is used here. There is no need for django.test dependency. 
I am sorry, I should have been more precise.

Reply:
https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client.